### PR TITLE
Awesome books objects

### PR DESCRIPTION
**Pull request by zeff96**
- Add Linters for HTML/CSS.
- Use best practices for HTML & CSS.
- Update README file.
- Add Mark up to the .html file.
- Add style to the website.
- Add Form to save books.
- use Local storage to save user data.
- Users can add books and can remove them from the list.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  saveBooks();
+};
+
+const deleteBooks = (idToDelete) => {
+  books = books.filter((book) => {
+    if (book.id === idToDelete) {
+      return false;
+    }
+    return true;
+  });
```
Comment (laiifuu) [inline index.js]: [OPTIONAL] Nice work on the book removing functionality :+1:. To make your code shorter here, I suggest you do the following: 
`books = books.filter((book) => book.id !== idToDelete);`

File: README.md [hunk 2]
```diff
@@ -1,288 +1,76 @@
...
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
 
-OPTIONAL SECTIONS:
-- FAQ
```
Comment (laiifuu) [inline README.md]: As you can see in this comment, all of what has been listed is a required field in the README except for the FAQ section. So, I suggest that you stick to the original template, and make adjustments by adding the information related to your project.

**Review by laiifuu — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle: 

## Hi team :wave: 


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
- [x] Great work on implementing the functionality :fire: 

## Required Changes ♻️
_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_Check the comments under the review :point_down:_

Cheers and :computer: Happy coding :computer: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @zeff96 and @MostafaShahat2021 , 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**